### PR TITLE
Feature/doc for 1.3

### PR DESCRIPTION
**Pull request by gangtao**
Updating document and test for 1.3 release prepare

### CODE REVIEW

File: python/timeplus/version.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.3.0b3"   # noqa: F401
```
Comment (jovezhong) [inline python/timeplus/version.py:1]: do we still want to call it b4?

**Review by jovezhong — APPROVED** [review]

File: python/timeplus/version.py (hunk 1 above)
Comment (gangtao) [reply python/timeplus/version.py:1]: the version will be changed automatically after I create new tag

**Comment by jovezhong** [comment]
(Jove Github Bot) added it to the current sprint.

**Comment by jovezhong** [comment]
(Jove Github Bot) moved this ticket out of the GitHub project(up to 1200 tickets for one project).

**Pull request merged**